uint96.go

```diff
@@ -0,0 +1,97 @@
+package ksuid
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/bits"
+)
+
+// uint96 represents an unsigned 96 bits little endian integer.
+type uint96 [3]uint32 // [0] holds low 32 bits, [1] holds middle 32 bits, [2] holds high 32 bits
+
+func uint96Payload(ksuid KSUID) uint96 {
+	return makeUint96FromPayload(ksuid[timestampLengthInBytes:])
+}
+
+// uint32(low): Takes the lowest 32 bits of the low uint64
+// uint32(low >> 32): Shifts the low value right by 32 bits and takes the result, giving us the middle 32 bits
+func makeUint96(high uint32, low uint64) uint96 {
+	return uint96{
+		uint32(low),       // lowest 32 bits
+		uint32(low >> 32), // middle 32 bits
+		high,              // highest 32 bits
+	}
+}
+
+func makeUint96FromPayload(payload []byte) uint96 {
+	return uint96{
+		binary.BigEndian.Uint32(payload[8:]),  // low (4 bytes)
+		binary.BigEndian.Uint32(payload[4:8]), // middle (4 bytes)
+		binary.BigEndian.Uint32(payload[:4]),  // high (4 bytes)
+	}
+}
```

Comment:
The comment above says _uint96 represents an unsigned 96-bit little-endian integer_. However, it expects a big-endian-encoded 12-byte slice, and the first 4 bytes (payload[:4]) represent the most significant part. and we are mapping these to v[2], v[1], and v[0] in that order will store them as a little-endian number internally (i.e., v[0] ends up holding the least significant 32 bits). Technically, this is correct, and there is no bug, but it’s confusing because everywhere else, we output big endian values (in `String`, `bytes`). Just make it clear that little-endian representation is purely internal.

Reply:
sure, I have updated the description.